.github/workflows/build.yaml

```
@@ -48,10 +48,10 @@ jobs:
       - name: Prepare manifests for linting
         run: |
           mkdir manifests
-          go run deploy/main.go my-images v0.0.8 vanilla > manifests/vanilla.yaml
-          go run deploy/main.go my-images v0.0.8 ocp > manifests/ocp.yaml
-          go run deploy/main.go my-images v0.0.8 vanilla my-secret > manifests/vanilla-with-secret.yaml
-          go run deploy/main.go my-images v0.0.8 ocp my-secret > manifests/ocp-with-secret.yaml
+          go run deploy/*.go --k8s-flavor vanilla my-images > manifests/vanilla.yaml
+          go run deploy/*.go --k8s-flavor ocp my-images > manifests/ocp.yaml
+          go run deploy/*.go --k8s-flavor vanilla --secret my-secret my-images > manifests/vanilla-with-secret.yaml
+          go run deploy/*.go --k8s-flavor ocp --secret my-secret my-images > manifests/ocp-with-secret.yaml
```

How about building once and running same binary?
```suggestion
          go build ./....
          ./image-prefetcher --k8s-flavor vanilla my-images > manifests/vanilla.yaml
          ./image-prefetcher --k8s-flavor ocp my-images > manifests/ocp.yaml
          ./image-prefetcher --k8s-flavor vanilla --secret my-secret my-images > manifests/vanilla-with-secret.yaml
          ./image-prefetcher --k8s-flavor ocp --secret my-secret my-images > manifests/ocp-with-secret.yaml
```